content/doc/developer/handling-requests/json.adoc

```diff
@@ -0,0 +1,271 @@
+---
+title: Expose HTTP API with json content with Jenkins
```

Comment:
```suggestion
title: Expose HTTP API with JSON content with Jenkins
```

[JSON](https://en.wikipedia.org/wiki/JSON) is an acronym (and thus spelled in uppercase letters)